graphqlapi-scm-1.rockspec

```diff
@@ -17,6 +17,7 @@ dependencies = {
     'http ~> 1',
     'errors ~> 2',
     'vshard ~> 0.1',
+    'cartridge ~> 2',
```

Comment:
I'd like to change both commit message and commit body

```
rockspec: add missing dependency

`cartridge` module is unconditionally required when the
`graphqlapi` entrypoint is required, so it should be a rock dependency.
```

And, please, remove cartridge install from deps and other places.

Reply:
Done

Reply:
Please, remove cartridge install from deps and other places.